### PR TITLE
0x59 msize

### PR DESCRIPTION
**Pull request by danilowhk**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

- 0x59 MSIZE opcode
-
-

### CODE REVIEW

File: src/kakarot/instructions/memory_operations.cairo [hunk 1]
```diff
@@ -83,4 +85,29 @@ namespace MemoryOperations {
...
+    // @return Updated execution context.
+    func exec_msize{
+        syscall_ptr: felt*,
+        pedersen_ptr: HashBuiltin*,
+        range_check_ptr,
+        bitwise_ptr: BitwiseBuiltin*,
+    }(ctx: model.ExecutionContext*) -> model.ExecutionContext* {
+        alloc_locals;
+        %{ print("0x59 - MSIZE") %}
+        let msize = Helpers.to_uint256(ctx.memory.raw_len);
```
Comment (LucasLvy) [inline src/kakarot/instructions/memory_operations.cairo:103]:
```suggestion
        let len = Memory.len(ctx.memory);
        let msize = Helpers.to_uint256(len);
```

Comment (LucasLvy) [reply src/kakarot/instructions/memory_operations.cairo:103]: raw len is the number of felts in the memory, as Uint256 is a struct composed by 2 felts raw_len = 2 * actual_len.

Comment (danilowhk) [reply src/kakarot/instructions/memory_operations.cairo:103]: Oh Ok! Thank You!

**Review by LucasLvy — APPROVED** [review]

**Comment by LucasLvy** [comment]
Lgtm

**Pull request merged**